4.0/gen-samples

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python2
+
+# Usage: get-conf-item item
+
+import glob
+try:
+    from vdsm.config import config
+except:
+    from six.moves import configparser
+    config = configparser.ConfigParser()
+
+config.read('pod_definition.conf')
+try:
+    cluster_name = config.get('vars', 'CLUSTER_NODE_NAME')
+    engine_fqdn = config.get('vars', 'ENGINE_FQDN')
+    nfs_base_path = config.get('vars', 'NFS_BASE_PATH')
+    nfs_server_address = config.get('vars', 'NFS_SERVER_ADDRESS')
+    spice_proxy_ip = config.get('vars', 'SPICE_PROXY_IP')
+    engine_password = config.get('vars', 'ENGINE_PASSWORD')
+    postgres_password = config.get('vars', 'POSTGRES_PASSWORD')
+except:
+    print 'Items are not configured properly'
+
+
+def gen_file(in_file):
+    filedata = None
+    with open(in_file, 'r') as file:
+        filedata = file.read()
+        filedata = filedata.replace('@CLUSTER_NODE_NAME@', cluster_name)
```

Comment:
And with @[]@ we can work with autotools.. wasn't should what we prefer here. I don't mind to change it and my gen script is not the best solution here of course but helps me to write a guide for this setup